src/mendeleyMain.vb

```diff
@@ -692,6 +692,33 @@ EndOfSub:
     uiDisabled = False
 End Sub
 
+Sub Remove_Bookmark()
+On Error GoTo ErrorHandler
```

Comment:
Question: because Remove_Bookmark() is called from insertCitationButton() and this oen is already setting up the ErrorHandler: is it needed here? I don't know if the ErrorHandler is propagated into other functions or needs to be set again.

And if needs to be done please do like everywhere else:
    If Not DEBUG_MODE Then
        On Error GoTo ErrorHandler
    End If

So we have the same type of code.